lib/manager/helmv3/artifacts.ts

```diff
@@ -0,0 +1,76 @@
+import { quote } from 'shlex';
+import { logger } from '../../logger';
+import { ExecOptions, exec } from '../../util/exec';
+import {
+  getSiblingFileName,
+  getSubDirectory,
+  readLocalFile,
+  writeLocalFile,
+} from '../../util/fs';
+import { UpdateArtifact, UpdateArtifactsResult } from '../common';
+
+async function helmUpdate(manifestPath: string): Promise<void> {
+  const cmd = `helm dependency update ${quote(getSubDirectory(manifestPath))}`;
```

Comment:
I think we need to add required repos before update? or will helm pull the required repos automatically?

Reply:
helmv3 handles this fine, it doesn't need any repositories configured

```
(/)m@omg:~/go/src/github.com/mikebryant/renovate-test-helm-lock/traefik$ helm repo list
Error: no repositories to show
(/)m@omg:~/go/src/github.com/mikebryant/renovate-test-helm-lock/traefik$ helm dependency update
Saving 1 charts
Downloading traefik from repo https://helm.traefik.io/traefik
Deleting outdated charts
(/)m@omg:~/go/src/github.com/mikebryant/renovate-test-helm-lock/traefik$ 
```